include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -61,7 +61,7 @@ __enable_if_t<
     sizeof(_Dst) == sizeof(_Src) && ::std::is_trivially_copyable_v<_Dst> && ::std::is_trivially_copyable_v<_Src>, _Dst>
 __dpl_bit_cast(const _Src& __src) noexcept
 {
-#if SYCL_LANGUAGE_VERSION >= 2020
```

Comment:
Should it be like we try to use std:: API first and only if it's not available try to use sycl?

Reply:
For as long as it is within the SYCL backend, I think it is more appropriate to use the SYCL version first.